webpack.config.js

```diff
@@ -2,7 +2,7 @@ var path = require('path');
 var NodePolyfillPlugin = require('node-polyfill-webpack-plugin');
 
 module.exports = {
-    target: ['web', 'es5'],
+    target: ['web', 'es2016'],
```

Comment:
Thanks very much for the PR.
We are moving our build processes from `webpack` to `esbuild` in #6909.
For this PR I suggest you revert this change and instead try adding `maplibre-gl` to the `module.rules[1].include` to be `buffer|d3-color|d3-interpolate|is-mobile|maplibre-gl`.

Reply:
I reverted this, and then the plotly overlays are working again (!)

Reply:
Yes. That webpack config appears to break maplibre.
So let's ignore the `no-es6` test for now.